Vagrantfile

```diff
@@ -1,36 +1,22 @@
 # -*- mode: ruby -*-
 # vi: set ft=ruby :
 
-# needed for the hyper-v stuff
-Vagrant.require_version ">= 1.7.4"
+Vagrant.require_version ">= 2.0.1"
```

Comment:
Ubuntu has 1.8.1 in the official package repositories for the 16.04 LTS version and 1.9.1 for the most recent version. This would require linux users to install third party packages. Personally, I don't mind that much, but other people might...

Reply:
oh wow, 1.9.1 is over a year old, and that's in 17.10...

anyways, i had 1.9.4 installed and that had a bug (https://github.com/hashicorp/vagrant/issues/8520) which destroyed the box directly after creation. it does not occur in 1.9.3 and 1.9.5. i simply didn't care and required the newest version.

i'm a bit unwilling to re-test this with older vagrant versions. how hard is it to install a debian package on ubuntu? (or to install the newest vagrant version by any other means)

Reply:
on both ubuntu and debian `apt-get install vagrant` should do the trick.

Reply:
Are you aware of any new feature that is not in 1.8.1 that you require? If not, I'd test whether it works with 1.8.1.

Reply:
It works fine with vagrant 1.8.1 on my machine. I also get the progress bar printed on multiple lines, but I don't care about that. Could you change the Vagrantfile and set the required vagrant version to 1.8.1?